bindings/alicloud/oss/oss.go

```diff
@@ -50,7 +50,11 @@ func (s *AliCloudOSS) Init(metadata bindings.Metadata) error {
 	return nil
 }
 
-func (s *AliCloudOSS) Write(req *bindings.WriteRequest) error {
+func (s *AliCloudOSS) Operations() []bindings.OperationKind {
+	return []bindings.OperationKind{bindings.CreateOperation}
+}
+
+func (s *AliCloudOSS) Invoke(req *bindings.InvokeRequest) (*bindings.InvokeResponse, error) {
 	key := ""
 	if val, ok := req.Metadata["key"]; ok && val != "" {
```

Comment:
Don't we need to check the OperationKind in the Invoke method? Why are we assuming that Invoke is for Write?